src/pages/docs/chat/moderation/direct/tisane.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Tisane
+meta_description: "Detect and remove unwanted content in a Chat Room using Tisane AI."
+---
+
+[Tisane](https://tisane.ai/) is a powerful Natural Language Understanding (NLU) platform that can be used to moderate content in chat rooms.
+
+The Tisane rule is a rule applied to chat rooms in Ably Chat which enables you to use Tisane's text moderation capabilities to detect and handle inappropriate content before it is published to other users.
```

Comment:
```suggestion
The Tisane integration can be applied to chat rooms so that you can use Tisane's text moderation capabilities to detect and handle inappropriate content before it's published to other users.
```